src/store/modules/stock/mutations.ts

```diff
@@ -0,0 +1,19 @@
+import { MutationTree } from 'vuex'
+import StockState from './StockState'
+import * as types from './mutation-types'
+
+const mutations: MutationTree <StockState> = {
+  [types.STOCK_PRODUCTS_UPDATED] (state, payload) {
+    payload.forEach((product: any) => {
+      if (product.sku) {
```

Comment:
I think this should never be empty for our case

Reply:
Removed the check.